tests/components/reolink/test_config_flow.py

```diff
@@ -481,4 +482,58 @@ async def test_https_repair_issue(hass: HomeAssistant) -> None:
     await hass.async_block_till_done()
 
     issue_registry = ir.async_get(hass)
-    assert len(issue_registry.issues) == 1
+    assert (const.DOMAIN, "https_webhook") in issue_registry.issues
+
+
+async def test_no_firmware_repair_issue(hass: HomeAssistant) -> None:
```

Comment:
Why is this in `test_config_flow.py` ?
I'm pretty sure it should be moved to `test_init.py`.

Reply:
@epenet Thanks for the suggestion, you are right.
I have split out the tests and moved the repair issue tests to a new test_init.py file.
Do you have any other feedback?

Reply:
Aha... Now I see that potential there is more cleanup that I could request.
I think you should create a preliminary PR to split out the existing methods out of `test_config_flow.py`:
- `test_config_flow.py` should have automatic patching of `async_setup_entry` with `pytestmark` (see https://github.com/home-assistant/core/pull/88951/files)
- I'm not sure `reolink_connect` should be auto-use. Maybe it should by auto-use only inside `test_init.py` (with pytestmark)
- `homeassistant/components/reolink/__init__.py` should be removed from `.coveragerc`

Reply:
@epenet the PR for cleaning up the reolink tests you requested is here: https://github.com/home-assistant/core/pull/89112 could you take a look at that PR.